modules/rte_mis_core/src/Form/OverrideTermForm.php

```diff
@@ -81,4 +81,39 @@ public function customSubmit(array $form, FormStateInterface $form_state) {
 
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
+    $term = $this->entity;
+
+    $result = $term->save();
+    $view_link = $term->toLink()->toString();
+
+    $this->messenger()->deleteAll();
+    $bundle = $this->entity->bundle();
+
+    $bundle = str_replace('_', ' ', $bundle);
+    $bundle = ucwords($bundle);
```

Comment:
Can we call the parent::submit and let parent method handle the save functionality, and use this save method to override the message only?

Reply:
Ok